src/Raven.Client/ServerWide/DatabaseRecord.cs

```diff
@@ -534,15 +535,57 @@ public override int GetHashCode()
             hash = 31 * hash + CompressAllCollections.GetHashCode();
             return hash;
         }
-        
+
         public DynamicJsonValue ToJson()
         {
             return new DynamicJsonValue
             {
-                [nameof(Collections)] =  new DynamicJsonArray(Collections),
+                [nameof(Collections)] = new DynamicJsonArray(Collections),
                 [nameof(CompressAllCollections)] = CompressAllCollections,
                 [nameof(CompressRevisions)] = CompressRevisions
             };
         }
     }
+
+    public static class DatabaseVersion
+    {
+        public const long BaseVersion = 40_000;
```

Comment:
Why 40_000 is the base version?

Reply:
Will we have a distinction in the base version between 5.4 and 6.0? Do we need that? If yes then we'll need to do similar checks like in `IndexVersion`:

https://github.com/ravendb/ravendb/blob/ab89ac5393770399d12b88e1e9bbebe23c8a36f3/src/Raven.Server/Documents/Indexes/IndexDefinitionBaseServerSide.cs#L184-L192

Reply:
changed :)